include/zephyr/drivers/timer/ifx_tcpwm.h

```diff
@@ -147,9 +152,10 @@
 	Cy_TCPWM_PWM_SetPeriod0((TCPWM_Type *)ADDRESS, 0, PERIOD0)
 #define IFX_TCPWM_PWM_SetPeriod1(ADDRESS, PERIOD1)                                                 \
 	Cy_TCPWM_PWM_SetPeriod1((TCPWM_Type *)ADDRESS, 0, PERIOD1)
+#define IFX_TCPWM_TriggerCaptureOrSwap_Single(ADDRESS)                                             \
+	Cy_TCPWM_TriggerCaptureOrSwap_Single((TCPWM_Type *)ADDRESS, 0)
 #define IFX_en_tcpwm_status_t(ADDRESS, MODE, PERIOD, DUTY, LIMITER)                                \
 	cy_en_tcpwm_status_t Cy_TCPWM_PWM_Configure_Dithering((TCPWM_Type *)ADDRESS, 0, MODE,      \
+/** @endcond */
```

Comment:
Likely meant this to be on the line after, otherwise you are cutting your multiline macro here